packages/facteur/facteur.0.1.0/opam

```diff
@@ -0,0 +1,52 @@
+opam-version: "2.0"
+license:      "MIT"
+authors:      [ "Romain Calascibetta <romain.calascibetta@gmail.com>" ]
+maintainer:   [ "Romain Calascibetta <romain.calascibetta@gmail.com>" ]
+homepage:     "https://github.com/dinosaure/facteur"
+bug-reports:  "https://github.com/dinosaure/facteur/issues"
+dev-repo:     "git+https://github.com/dinosaure/facteur.git"
+doc:          "https://dinosaure.github.io/facteur/"
+synopsis:     "Tool to send an email"
+description: """Tool to send an email in OCaml according RFC4409 [SUBMIT]
+"""
+
+build: [ "dune" "build" "-p" name "-j" jobs ]
+run-test: [ "dune" "runtest" "-p" name "-j" jobs ]
+
+depends: [
+  "ocaml" {>= "4.03.0"}
+  "mrmime"
+  "colombe"
+  "sendmail-lwt"
+  "logs"
+  "fmt"
+  "astring"
+  "fpath"
+  "cmdliner"
+  "domain-name" {>= "0.3.0"}
+  "x509" {>= "0.7.0"}
+  "emile" {>= "0.5"}
+  "dune" {>= "1.3"}
+]
+
+depexts: [
+  ["file-dev"] {os-distribution = "alpine"}
+  ["file-devel"] {os-distribution = "centos"}
+  ["file-devel"] {os-distribution = "fedora"}
+  ["file-devel"] {os-family = "suse"}
+  ["libmagic-dev"] {os-family = "debian"}
+  ["libmagic"] {os = "macos" & os-distribution = "homebrew"}
+]
+
+conflicts: [
```

Comment:
Why are those conflicts when they already are hard dependencies. Could you just add the constraint `{>= "0.1.0"}` directly to the dependencies above?